src/c.jl

```diff
@@ -47,22 +47,40 @@ struct xls_summaryInfo
     company::Cstring
 end
 
+struct st_cell_data
+    id::UInt16
+    row::UInt16
+    col::UInt16
+    xf::UInt16
+    str::Ptr{UInt8} # Cstring
```

Comment:
Can't we declare this as `Cstring` right away?

Reply:
Yes! Just tested and works fine.